src/index.ts

```diff
@@ -188,6 +188,24 @@ export class KhanApi {
     return this.graphQL("/getStudentsList", payload)
   }
 
+  getClassroomRoster = async ({
+    classId = "",
+    pageSize = 40,
+    teacherKaid,
+  }: {
+    classId?: string
+    pageSize?: number
+    teacherKaid?: string
+  }) => {
```

Comment:
```suggestion
  getClassroomRoster = async ({
    classId,
    teacherKaid,
    pageSize = 40
  }: {
    classId: string
    teacherKaid: string
    pageSize?: number
  }) => {
```

I think we want both the `classId` and the `teacherKaid` to be required based on the query, i.e. it requires `String!`.

Reply:
Agreed